src/command/add.rs

```diff
@@ -3,9 +3,11 @@ use std::collections;
 use std::fs::{self, File};
 use std::io::{self, Read, Write};
 use std::os::linux::fs::MetadataExt;
+use std::os::unix::ffi::OsStrExt;
 
 use byteorder::{BigEndian, ByteOrder};
 use hex;
+use std::path::{Path, PathBuf};
```

Comment:
```suggestion
use std::os::unix::ffi::OsStrExt;
use std::path::{Path, PathBuf};

use byteorder::{BigEndian, ByteOrder};
```

個人的な好みと、`use hex;`はなくていいはず(未確認)